Makefile

```diff
@@ -24,8 +24,9 @@ all: build
 .PHONY: build
 build: | $(ENV)/COMPLETE
 $(ENV)/COMPLETE: requirements.txt
-	$(VIRTUALENV) $(ENV)
+	$(VIRTUALENV) $(ENV) --no-setuptools
 	$(INSTALL) -i https://pypi.python.org/simple -U pip
+	$(INSTALL) -U "setuptools<45"
```

Comment:
Please add a comment above this line to briefly explain that this version of setuptools is incompatible with python2.

Also, if you think this could be simplified by just deleting the `-U pip` line above and using the version of pip installed by virtualenv, please do try it out. Opportunistically updating to the latest versions was a good idea years ago but seems like a bad idea now that we're playing the legacy-support game.